app/docs/0.10.x/secure-admin-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Securing the Admin API
+---
+
+# Overview
+
+Kong's Admin API provides a RESTful interface for administration and
+configuration of APIs, plugins, consumers, and credentials. Because this
+API allows full control of Kong, it is important to secure this API against
+unwanted access. This document describes a few approaches to take to secure
+the Admin API.
```

Comment:
"approaches to take to secure" not native speaking, but looks a bit awkward  to me? please ignore this comment if not